### ISSUE TITLE
ion-item-divider is not compliant with dark theme

### ISSUE THREAD

**Issue by msegers** [driftyco/ionic#7867]
#### Short description of the problem:

In the dark theme ionic-item-divider has a white background.
This also differs per theme (md ios wp). With Windows phone you get white text on a white bg by default.
#### What behavior are you expecting?

A darker colour for the divider

**Steps to reproduce:**
1. create a list, and add the ion-item-divider
2. set up the project to a black theme by importing this in variables.scss
3. run and hide or inspect

**Which Ionic Version?** 
2 beta 11
#### Plunker that shows an example of your issue

**For Ionic 1 issues** - http://plnkr.co/edit/Xo1QyAUx35ny1Xf9ODHx?p=preview

**For Ionic 2 issues** - http://plnkr.co/edit/me3Uk0GKWVRhZWU0usad?p=preview

**Run `ionic info` from terminal/cmd prompt:** (paste output below)

**Comment by brandyscarney** [comment]
How are you importing the dark theme? The styles for the theme are in each individual theme folder, so adding it in each mode specific sass file works. For example, in the `app.wp.scss` file:

```
// Ionic Windows Sass
// --------------------------------------------------
// Custom App variables must be declared before importing Ionic.
// Ionic will use its default values when a custom variable isn't provided.
@import 'themes/dark.wp';
@import "ionic.wp";
```

However, if you are importing only the dark Sass file it will show white dividers.

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!

**Comment by msegers** [comment]
@brandyscarney Hi Brandy,

I was actually adding it in my webpack.config dynamically, so pretty nasty I recon. But I was trying to set it up with a dynamic configuration. I'm using the same application for multiple apps and potentially I would be using the dark theme in the future. I refrained from it now since it seems it doesn't work (in my use case).

```
sassLoader: {
        includePaths: [
            'node_modules/ionic-angular',
            'node_modules/ionicons/dist/scss'
        ],
        data: "$custom-primary: " + config.colorPrimary + " ; $custom-accent: " + config.colorAccent + " ; " + (config.darkTheme ? "@import '../../node_modules/ionic-angular/themes/dark';" : ""),
    },
```

The custom-primary and custom-accent are my way of overriding the variables for each flavour as of now. I couldn't find any better solution that this which is slightly hacky but it works decently for me.

I guess at this point I'm probably importing the dark theme too early, however most of it seems to work. If you have any thoughts on this I'd like to hear them otherwise closing is fine and I'll just create 6 css files instead, android-dark, ios-dark wp-dark and regular.

Apoligies for the late response but sometimes I get too many Github mails and those are hard to follow in the weekends ;-).

**Comment by brandyscarney** [comment]
@msegers I know it's been awhile since this was addressed - but are you still seeing this issue with your configuration or did you get it figured out?

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!

**Comment by WillGeller** [comment]
I am not sure if this has changed I am on the latest Ionic 3.6.0 and am using the dark theme by replacing the default import in variables.scss as so:

> // App Theme
//@import "ionic.theme.default";
@import "ionic.theme.dark";

For the most part the dark theme looks good but I did have to add to app.scss:

> h1, h2, h3, h4, p {
  color: white !important;
}

And the most noticeable problem is the **alerts** are still styled with white background dark text.

Wondering if there is something else I should do to enable the dark theme before adding more custom CSS

**Comment by msegers** [comment]
I'm no longer using iOnic I'm afraid, this was when themes were in beta anyway I believe.
